src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -651,6 +651,24 @@ def testZeroLengthFiles(self):
                 self.assertEquals(f.read(), "")
             self.master.delete(job.jobStoreID)
 
+        def testGrowingAndShrinkingJob(self):
```

Comment:
This is great.

Reply:
It does waste 50MB of upload/download, though. I could shrink it down to whatever the minimal amount is that still triggers the overlarge job code.

Reply:
I think it's 5MB, though looking at the helper, I think 200kb should trigger it.

Reply:
D'oh, you're totally right. I shifted it down to 500kB, because the Azure jobStore only allows jobs that are less than 1MB. (Which means we eventually need to add overlarge jobs to Azure, too... but one step at a time.)

Reply:
That's a good idea.  And Azure will get there.  I'll file a ticket so that I don't forget.